Dockerfile

```diff
@@ -5,6 +5,8 @@ WORKDIR /home/node/app
 COPY package*.json /home/node/app/
 COPY . /home/node/app/
 
+RUN npm install -g npm@10.8.1
```

Comment:
why is this needed?

Reply:
Otherwise it always complains about a missing upgrade.